src/GameSync.Api.Integration/GameSyncService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using GameSync.Models.Users;
+using GameSync.Models.Users.ValueObjects;
+
+namespace GameSync.Api.Integration;
+public class GameSyncService : IGameSyncService
+{
+    public User Login(UserName userName, Password password)
+    {
+        if (userName.Value == "admin" &&  password.Value == "adminadmin")
+        {
+            return new User(
+                UserId.CreateUnique(),
```

Comment:
I suppose this is for demo purposes, right?

Reply:
Yes. Only for demo